test/index_tests.js

```diff
@@ -37,6 +37,8 @@ it('WHEN patch function is provided THEN it is applied to spec', done => {
       request.get(`http://localhost:${port}${path}?a=1`, () => {
         const spec = generator.getSpec();
         expect(spec.info.title).toBe(newTitle);
+        expect(spec.info.description).toContain('(/api-spec/v2)');
```

Comment:
Same here: `versions.OPEN_API_V2` + `versions.OPEN_API_V3`. Think we need to export versions from `lib/openapi.js `on the test file.

Reply:
fixed